lib/http.js

```diff
@@ -6,9 +6,43 @@ const EventEmitter = require('events').EventEmitter;
 
 exports.IncomingMessage = require('_http_incoming').IncomingMessage;
 
-
-const common = require('_http_common');
-exports.METHODS = common.methods.slice().sort();
+// Request Methods
+var METHODS = [
+  'DELETE',
+  'GET',
+  'HEAD',
+  'POST',
+  'PUT',
+  // pathological
+  'CONNECT',
+  'OPTIONS',
+  'TRACE',
+  // webdav
+  'COPY',
+  'LOCK',
+  'MKCOL',
+  'MOVE',
+  'PROPFIND',
+  'PROPPATCH',
+  'SEARCH',
+  'UNLOCK',
+  // subversion
+  'REPORT',
+  'MKACTIVITY',
+  'CHECKOUT',
+  'MERGE',
+  // upnp
+  'MSEARCH',
+  'NOTIFY',
+  'SUBSCRIBE',
+  'UNSUBSCRIBE',
+  // RFC-5789
+  'PATCH',
+  'PURGE'
+];
+exports.__defineGetter__('METHODS', util.deprecate(function() {
```

Comment:
Not trying to encourage it, but at this point, the most likely outcome is that defineGetter and friends will remain (as a method on `Object.prototype`). 

https://github.com/rwaldron/tc39-notes/blob/07a99722d9f2594a51fd62e9ea320d72caabc01a/es6/2014-01/jan-30.md#definegetter-definesetter-etc-in-annex-b

Object.defineProperty would feel cleaner.